apps.json

```diff
@@ -146,7 +146,7 @@
     },
     "bitwarden": {
         "branch": "master",
-        "level": 7,
+        "level": 0,
```

Comment:
For bitwarden, I have an ongoing [PR](https://github.com/YunoHost-Apps/bitwarden_ynh/pull/14), if we can wait a bit. 
 :)

Reply:
Meh, failed again

Reply:
Let's merge, I will work on it.